components/heatmap/HeatMap.vue

```diff
@@ -221,7 +221,11 @@ export default {
     }),
     ...mapState("playback", ["playback_state"]),
     metric_names: function () {
-      return Object.keys(this.well_values);
+      const temp = Object.keys(this.well_values);
```

Comment:
I would keep this as is and instead swap the keys in  `heatmap_values` in  `store/modules/data/index.js` and you'll want to update the ranges I think, it's still 100